Mention optional attributes for CloudEvents in event sources overview (#4627)
knative/docs: docs/eventing/sources/README.md

    kubectl get sources
    ```

> Please note: Event Sources which import events from other messaging technologies (such as Kafka or Rabbit) are not responsible for setting [Optional Attributes](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#optional-attributes) such as the `datacontenttype`. This is considered a responsibility of the original event producer, and the Source will simply append attributes if they exist.

Reviewer: ```suggestion
!!! note
    Event Sources that import events from other messaging technologies such as Kafka or RabbitMQ are not responsible for setting [Optional Attributes](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#optional-attributes) such as the `datacontenttype`. This is a responsibility of the original event producer; the Source just appends attributes if they exist.
```
